PowerShellTools.HostService/ServiceManagement/Debugging/PowershellDebuggingServiceUtilities.cs

```diff
@@ -103,8 +103,18 @@ private void DebuggerFinished()
         {
             ServiceCommon.Log("DebuggerFinished");
 
-            ClearBreakpoints();
-            _psBreakpointTable.Clear();
+            try

```

Comment:
I thought by wrapping up the v4+ api into separate method would just solve the problem, if we do want to make the clearbreakpoint not throw, I would do that(try/catch) inside clearbreakpoint function itself.